### PR TITLE
Allow GitHub Container Registry

### PR DESCRIPTION
**Pull request by docjyJ**

### CODE REVIEW

**Comment by Zoey2936** [comment]
if I remember correctly, ghcr is very slow

**Comment by docjyJ** [comment]
> if I remember correctly, ghcr is very slow

I don't know, but is this a problem?

I find that GitHub is more practical (no secret, only one account, integration with sources).

File: php/containers-schema.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 					"image": {
 						"type": "string",
 						"minLength": 1,
-						"pattern": "^[a-z0-9/-]+$"
+						"pattern": "^(ghcr[.]io[/])?[a-z0-9-]+[/][a-z0-9-]+$"
```
Comment (szaimen) [inline php/containers-schema.json:18]: The reason that we dont allow ghcr.io is that we could not make the container update check work with ghcr.io https://github.com/nextcloud/all-in-one/blob/main/php/src/Docker/DockerHubManager.php

Comment (szaimen) [reply php/containers-schema.json:18]: If someone finds a solution then we can allow it :)

**Pull request closed without merging**